doc/user-guide.md

```diff
@@ -58,11 +58,20 @@ By default, the memcached-operator watches `Memcached` resource events as shown
 
 ```Go
 func main() {
+  // Default number of worker to 1
   sdk.Watch("cache.example.com/v1alpha1", "Memcached", "default", 5)
   sdk.Handle(stub.NewHandler())
   sdk.Run(context.TODO())
 }
 ```
+```Go
+func main() {
+  // Allow for flexible number of workers (N) >= 1
+  sdk.Watch("cache.example.com/v1alpha1", "Memcached", "default", 5, sdk.WithNumWorkers(N))
+  sdk.Handle(stub.NewHandler())
+  sdk.Run(context.TODO())
+}
+```
```

Comment:
**Note:** You can configure the number of concurrent informer workers with an additional Watch option:
```Go
sdk.Watch("cache.example.com/v1alpha1", "Memcached", "default", 5, sdk.WithNumWorkers(n))
```

Reply:
Also mention that the default is 1 over here.